includes/templates/class-amp-post-template.php

```diff
@@ -267,23 +267,25 @@ private function build_post_data() {
 
 		$this->add_data(
 			array(
-				'post'                    => $this->post,
-				'post_id'                 => $this->ID,
-				'post_title'              => $post_title,
-				'post_publish_timestamp'  => $post_publish_timestamp,
-				'post_modified_timestamp' => $post_modified_timestamp,
-				'post_author'             => $post_author,
+				'post'                     => $this->post,
+				'post_id'                  => $this->ID,
+				'post_title'               => $post_title,
+				'post_publish_timestamp'   => $post_publish_timestamp,
+				'post_modified_timestamp'  => $post_modified_timestamp,
+				'post_author'              => $post_author,
+				'post_canonical_link_url'  => get_permalink( $this->ID ),
+				'post_canonical_link_text' => __( 'Exit Reader Mode', 'amp' ),
```

Comment:
Alternatively, we could say "View Canonical". What do you think?

Reply:
I don't think the average user knows what "Canonical" means in this context. Not to speak of translators. "Original" or "Full Version" would work better though

`Exit Reader Mode` sounds reasonable though.

For comparison, Safari uses "Hide Reader View".

Reply:
I see the point of "View Canonical", but it is true that is will be confusing for most users. OTOH, having "Exit Reader Mode" implies that the user is aware that she is in Reader Mode. Probably we can express the duality by a exit route with a message along these lines: "You are viewing the Reader Mode version of `<this post>`", where `<this post>` links to the canonical version.